setup.py

```diff
@@ -37,7 +37,9 @@
     "google-resumable-media >= 1.3.0, < 2.0dev; python_version<'3.0'",
     "google-resumable-media >= 1.3.0, < 3.0dev; python_version>='3.6'",
     "requests >= 2.18.0, < 3.0.0dev",
+    "protobuf < 3.18.0; python_version<'3.0'",
```

Comment:
Should we be adding an un-pinned `protobuf` for Python 3.x (to avoid future transitive dep failures):
```suggestion
    "protobuf < 3.18.0; python_version<'3.0'",
    "protobuf; python_version>='3.6'",
```